lib/klaviyo/apis/identify.rb

```diff
@@ -0,0 +1,23 @@
+module Klaviyo
+  class Identify < Client
+    def self.identify(kwargs = {})
+      defaults = {:id => nil, :email => nil, :properties => {}}
```

Comment:
can we rename things from called defaults or kwards and just make them more specific?

```suggestion
      defaults = {
          :id => nil,  # should we use klaviyo_id instead
          :email => nil, 
          :properties => {}
      }
```